.circleci/config.yml

```diff
@@ -8,6 +8,45 @@ orbs:
   slack: circleci/slack@4.9.3
 
 commands:
+  slack-basic:
+    parameters:
+      event:
+        description: when this notification will fire. options are fail, pass, always.
+        type: string
+        default: always
+      branch_pattern:
+        description: branch_pattern arg for slack/notify
+        type: string
+        default: .+
+      channel:
+        description: Slack channel to send message to
+        type: string
+        default: C03EK0C69QD
```

Comment:
can this be a context var?

Reply:
Good catch!

Updated to match the orb's envar.